boxwise_wms/tests/test_random_id_sequence.py

```diff
@@ -0,0 +1,73 @@
+from odoo.tests import common
+from odoo.exceptions import Warning as UserWarning
+import logging
+import re
+
+_logger = logging.getLogger(__name__)
+
+class TestRandomIdSequence(common.TransactionCase):
+    def setUp(self):
+        super(TestRandomIdSequence, self).setUp()
+
+    @common.post_install(True)
```

Comment:
According to the docs

> By default, tests are run once right after the corresponding module has been installed. 

and also for this post_install flag

> The flag is a boolean specifying whether the test should or should not run after a set of module installations.
By default, tests are not run after installation of all modules in the current installation set.

This seems to contradict - @DurkoMatko what was your understanding having experimented with this?